### ISSUE TITLE
Send arbitrary keyboard scans in boot_command property

### ISSUE THREAD

**Issue by mlenormand** [mitchellh/packer#810]
Not all keyboard scans can be sent to the VM : only a few special commands are supported (enter, esc, f1, ...).
I want to send some special commands (e.g. "top-oriented array" key) but it seems I can't.
Could it be possible to specify directly scancodes in the boot_command property, from http://www.win.tue.nl/~aeb/linux/kbd/scancodes-1.html ?

Go file : https://github.com/mitchellh/packer/blob/master/builder/virtualbox/iso/step_type_boot_command.go

Thanks in advance for your help !

**Comment by tamascsaba** [comment]
Hy!

How is it possible? 
```
        "boot_command" : [
            "<enter>",
            "<wait10>YES<enter><wait10><wait10><wait10>",
            "1d 38 53"
        ]
```
1d 38 53 => ctrl+alt+del keyboard scan to restart vm

**Comment by tamascsaba** [comment]
@mitchellh ???

**Comment by michaeltlombardi** [comment]
Should this have been closed by #3767?

**Comment by rickard-von-essen** [comment]
Mostly fixed by #3767. Open a new issue for cases that is not solved by it.
